generators/minimal-go/prompts.go

```diff
@@ -18,38 +17,33 @@ func getAnswers() (ProjectConfig, error) {
 }
 
 func getProjectConfigAnswers() (ProjectConfig, error) {
-	var questions = []*survey.Question{
-		{
-			Name: "AppName",
-			Prompt: &survey.Input{
-				Message: "What is the base name of your application?",
-				Help:    "Name of your application",
-				Default: "myapp",
-			},
-			Validate:  helpers.ValidateApplicationName,
-			Transform: helpers.TrimString,
-		},
-		{
-			Name: "ModulePath",
-			Prompt: &survey.Input{
-				Message: "What is your application module path?",
-				Help:    "Module path ex: github.com/username/modulename",
-				Default: "github.com/username/myapp",
-			},
-			Validate: survey.Required,
-		},
-		{
-			Name: "RoutingLibrary",
-			Prompt: &survey.Select{
-				Message: "Choose Routing Library:",
-				Options: []string{"Gin", "Chi"},
-				Default: "Gin",
-			},
-		},
-	}
+
 	answers := ProjectConfig{}
-	err := survey.Ask(questions, &answers)
-	if errors.Is(err, terminal.InterruptErr) {
+	form := huh.NewForm(
+		huh.NewGroup(
+			huh.NewInput().
+				Title("What is the base name of your application?").
+				Description("Name of your application").
+				Validate(func(str string) error {
+					return helpers.ValidateApplicationName(str)
+				}).
+				Value(&answers.AppName),
```

Comment:
To retain the existing behavior of providing "Default" values, we can initialize `answers := ProjectConfig{}` with default values.